web-server/v0.4/src/pages/ComparisonSelect/index.js

```diff
@@ -133,7 +135,15 @@ class ComparisonSelect extends React.Component {
               };
               return (
                 <div key={iteration.resultName} style={{ marginTop: 32 }}>
-                  <h2>{iteration.resultName}</h2>
+                  <h2>
+                    {iteration.resultName}{' '}
+                    <span>
+                      <Tag color="blue" style={{ fontSize: '70%' }}>
```

Comment:
Is there a specific reason we use percentage for fontSizes? I believe ant.design automatically scales fontSizes for their components and we usually use the default font sizes for all components within the dashboard.